(feat): add deployment page (#350)
chalk-ai/docs: gcp-cloud-deployment.mdx

import { ServiceDiagramSwitcher } from '@/components/ArchitectureDiagram'

> This page contains GCP specific information on Chalk deployment. A general

Author: ```suggestion
This page contains GCP specific information on deploying Chalk. A general
```

bigquery.models.updateTag
```

## Granting the Roles

> The following bash script can be used to grant all the roles to the chalk provided service account:

Author: ```suggestion
The following bash script can be used to grant the roles we need to deploy Chalk into your GCP project :
```

> The following bash script can be used to grant the roles we need to deploy Chalk into your GCP project :

Author: ```suggestion
The following bash script can be used to grant the permissions we need to deploy Chalk into your GCP project :
```

> The following bash script can be used to grant the permissions we need to deploy Chalk into your GCP project :

Author: ```suggestion
The following bash script can be used to grant the permissions we need to deploy Chalk into your GCP project:
```
